lemur/certificates/service.py

```diff
@@ -1240,3 +1241,19 @@ def is_valid_owner(email):
 
     # expecting owner to be an existing team DL
     return user_membership_provider.does_group_exist(email)
+
+
+def allowed_issuance_for_domain(common_name, extensions):
+    check_permission_for_cn = True if common_name else False
+
+    # authorize issuance for every x509.DNSName SAN
+    if extensions and extensions.get("sub_alt_names"):
+        for san in extensions["sub_alt_names"]["names"]:
+            if isinstance(san, x509.DNSName):
+                if san.value == common_name:
+                    check_permission_for_cn = False
+                is_authorized_for_domain(san.value)
```

Comment:
not returning the outcome of the AuthZ evaluation?

Reply:
It raises `UnauthorizedError` if authz fails ([additional details](https://github.com/Netflix/lemur/pull/3889/files#r766258425))